.ci/Jenkinsfile

```diff
@@ -190,26 +191,47 @@ pipeline {
             tag pattern: 'v\\d+.*', comparator: 'REGEXP'
           }
         }
-        steps {
-          withGithubNotify(context: 'Release') {
-            deleteDir()
-            unstash 'source'
-            script {
-              dir(BASE_DIR){
-                docker.image("${env.RUBY_DOCKER_TAG}").inside('-v /etc/passwd:/etc/passwd -v ${HOME}/.ssh:${HOME}/.ssh') {
-                  withEnv(["HOME=${env.WORKSPACE}/${env.BASE_DIR ?: ''}"]) {
-                    rubygemsLogin.withApi(secret: "${env.RELEASE_SECRET}") {
-                      sshagent(['f6c7695a-671e-4f4f-a331-acdce44ff9ba']) {
-                        sh '.ci/prepare-git-context.sh'
-                        sh 'gem install rake yard rspec'
-                        sh 'rake release'
+        stages {
```

Comment:
Created a top-level release stage which contains:
- a nested `release` stage as used to be -> https://github.com/elastic/apm-agent-ruby/pull/635/files#diff-8db37e8fcea0f1a8f2f39667e94ebcc4R195
- a new nested `opbeans` stage <--- this is the new thing. https://github.com/elastic/apm-agent-ruby/pull/635/files#diff-8db37e8fcea0f1a8f2f39667e94ebcc4R218